Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/urls.ts

```diff
@@ -244,22 +244,18 @@ const getDownloadProblemResourceUrl = ({ id }: IDownloadProblemResourceUrlParams
 const getHomeStatisticsUrl = () => `${baseApiUrl}/StatisticsPreview/GetForHome`;
 
 // Search
-const getSearchResults = ({ searchTerm, page, selectedTerms }: IGetSearchResultsUrlParams) => {
+const getSearchResultsUrl = ({ searchTerm, page, searchCategory }: IGetSearchResultsParams) => {
     const searchQuery = `${SearchParams.search}=${searchTerm}`;
 
     const pageQuery = `page=${page}`;
 
-    const selectedTermQuery = `&${selectedTerms
-        .map(({ key, value }) => `${key}=${value}`)
-        .join('&')
-    }`;
-
-    return `${baseApiUrl}/Search/GetSearchResults?${searchQuery}${selectedTermQuery}&${pageQuery}`;
+    return `${baseApiUrl}/Search/Get${searchCategory}SearchResults?${searchQuery}&${searchCategory}=true&${pageQuery}`;
```

Comment:
What if there is no marked category(empty). Then the queryString will be set ' '=true.

Reply:
I added a conditional rendering with a constant message for no search category being selected.